src/apps/dashboard-projects/List/ProjectEntry.tsx

```diff
@@ -2,7 +2,7 @@ import { useState } from 'react';
 import { joinProject } from '@backend/helpers';
 import { supabase } from '@backend/supabaseBrowserClient';
 import { Avatar } from '@components/Avatar';
-import { OwnerPill } from '@components/OwnerPill';
+import { LockedPill } from '@components/LockedPill';
```

Comment:
So we no longer use the "owner" pill in the list view? Or are we deprecating the owner pill altogether?

Reply:
Yes. I believe we decided that since all projects on the "My Projects" page are ones you own it did not make much sense.